cmd/gaia/init/gentx.go

```diff
@@ -129,13 +130,28 @@ following delegation and commission default parameters:
 				return err
 			}
 
+			info, err := txBldr.Keybase().Get(name)
+			if err != nil {
+				return err
+			}
+
+			if info.GetType() == kbkeys.TypeOffline {
+				fmt.Println("Offline key passed in.  Use `gaiacli tx sign` command to sign:")
+				return utils.PrintUnsignedStdTx(txBldr, cliCtx, []sdk.Msg{msg}, true)
+			}
+
 			// write the unsigned transaction to the buffer
 			w := bytes.NewBuffer([]byte{})
 			cliCtx = cliCtx.WithOutput(w)
+
 			if err = utils.PrintUnsignedStdTx(txBldr, cliCtx, []sdk.Msg{msg}, true); err != nil {
 				return err
 			}
 
+			if info.GetType() == kbkeys.TypeOffline {
```

Comment:
When would we hit this line - wouldn't we have returned on line 138?

Reply:
Oops, fixing.